Makefile

```diff
@@ -180,6 +180,9 @@ tests/%.prove: tests/%
 tests/%.klab-prove: tests/%
 	$(TEST) klab-prove --backend $(TEST_SYMBOLIC_BACKEND) $< --format-failures --def-module $(KPROVE_MODULE)
 
+tests/%.prove-repl: tests/%
+	$(TEST) prove --backend $(TEST_SYMBOLIC_BACKEND) --debug $< --format-failures --def-module $(KPROVE_MODULE)
+
```

Comment:
Instead of doing it this way, I would just call `make tests/my-spec.k.prove KPROVE_OPTIONS='--debug'`, and add a variable `KPROVE_OPTIONS` which can be fed into both the `%.prove` and `%.klab-prove` targets.

The problem with adding lots of new build targets is they can get out of sync. Your choice.

Reply:
Sounds like a good idea, I prefer that.